content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx

```diff
@@ -558,7 +558,7 @@ It returns [summaries](#chat-summaries) of the chats an Agent has access to.
 
 | Request object                                        | Required | Type     | Notes                                                                                       |
 | ----------------------------------------------------- | -------- | -------- | ------------------------------------------------------------------------------------------- |
-| `filters`                                             | No       | `object` | Mustn't change between requests for subsequent pages. Otherwise, the behavior is undefined. |
+| `filters`                                             | No       | `object` |                                                                                             |
```

Comment:
I would add the same note as in Web reference and ask someone from the API team to review it and let us know if it's clear.

Reply:
updated in f55e6c166737c4a8049546b77dd370d6a53e97d2